httpserver/server.go

```diff
@@ -84,38 +109,38 @@ func (server *Server) Run() error {
 		ReadHeaderTimeout: readHeaderTimeout,
 	}
 
-	if server.CertDirectory != "" {
-		certFile := filepath.Join(server.CertDirectory, certName)
-		keyFile := filepath.Join(server.CertDirectory, keyName)
-
-		logrus.Info(fmt.Sprintf("%s: [%s:%s] [%s:%s]", "starting server using TLS", "certFile", certFile, "keyFile", keyFile))
+	if server.CertDirectory == "" {
+		logrus.Info("starting server without TLS")
+		return svr.Serve(lsnr)
+	}
 
-		if server.CaCertFile != "" {
-			caCert, err := os.ReadFile(server.CaCertFile)
-			if err != nil {
-				panic(err)
-			}
+	certFile := filepath.Join(server.CertDirectory, certName)
+	keyFile := filepath.Join(server.CertDirectory, keyName)
 
-			clientCAs := x509.NewCertPool()
-			clientCAs.AppendCertsFromPEM(caCert)
+	logrus.Info(fmt.Sprintf("%s: [%s:%s] [%s:%s]", "starting server using TLS", "certFile", certFile, "keyFile", keyFile))
```

Comment:
use `Infof` ?

Reply:
These logs existed before. The reason it's using `Info` is the value is just a string with `fmt.Sprintf`, so there is no formatting required.